jekyll/_cci2/contexts.md

```diff
@@ -41,10 +41,12 @@ If you find you need to rename an org or repo that you have previously hooked up
 
 ## Creating and Using a Context
 
-1. As an organization member, navigate to the Organization Settings page in the CircleCI application, by clicking on the link in the sidebar. **Note:** Any organization member can create a context, but only organization administrators can restrict it with a security group.
+1. Using the new version of the CircleCI application, navigate to the Organization Settings page by clicking on the link in the sidebar. **Note:** Any organization member can create a context, but only organization administrators can restrict it with a security group.
     
     ![Contexts]({{ site.baseurl }}/assets/img/docs/org-settings-contexts-v2.png)
 
+  **If you are using the old version of the CircleCI application you can access the Organization Settings using the following URL format `https://circleci.com/gh/organizations/<org>/settings#contexts` substituting your organization/GitHub username. If you are using CircleCI Server Organization Settings can still be accessed as normal from the main navigation.
```

Comment:
```suggestion
  **If you are using the old version of the CircleCI application, you can access the Organization Settings using the following URL format `https://circleci.com/gh/organizations/<org>/settings#contexts`, substituting your organization/GitHub username. If you are using CircleCI Server Organization, Settings can still be accessed as normal from the main navigation.
```

Reply:
Thanks @thomasr8! I clearly had a punctuation bypass this afternoon!